data/json/recipes/ammo/components.json

```diff
@@ -320,5 +320,22 @@
     "qualities": [ { "id": "SAW_M", "level": 1 } ],
     "tools": [ [ [ "press", -1 ] ], [ [ "cordless_drill", 2 ], [ "corded_powerdrill", 2 ] ] ],
     "components": [ [ [ "460_casing", 1 ] ] ]
+  },
+  {
+    "type": "recipe",
+    "activity_level": "LIGHT_EXERCISE",
+    "result": "50ae_casing",
+    "category": "CC_*",
+    "subcategory": "CSC_*_NESTED",
+    "skill_used": "fabrication",
+    "difficulty": 2,
+    "skills_required": [ "gun", 3 ],
+    "time": "2 m 6 s",
+    "//": "Operations required: trimming, resizing.",
```

Comment:
Since we're going from a rifle casing to a pistol one, it's safe to assume that the rifle one is thicker internally than the pistol one.

I recommend using one of the recipes that has trimming, reaming, and resizing instead.